src/SecTester.Core/Logger/DefaultConsoleFormatterOptions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Extensions.Logging.Console;
+
+namespace SecTester.Core.Logger;
+
+public class DefaultConsoleFormatterOptions: ConsoleFormatterOptions

```

Comment:
Redundant class, you can remove it without any issues. Use `ConsoleFormatterOptions` instead